src/components/application_manager/test/commands/hmi/ui_get_capabilities_response_test.cc

```diff
@@ -267,6 +289,62 @@ TEST_F(UIGetCapabilitiesResponseTest, SetPhonenCapability_SUCCESS) {
   command->Run();
 }
 
+TEST_F(UIGetCapabilitiesResponseTest, SetVideoStreamingCapability_SUCCESS) {
+  MessageSharedPtr command_msg = CreateCommandMsg();
+  (*command_msg)[strings::msg_params][strings::system_capabilities] =
+      smart_objects::SmartObject(smart_objects::SmartType_Map);
+
+  (*command_msg)[strings::msg_params][strings::system_capabilities]
+                [strings::video_streaming_capability] =
+                    smart_objects::SmartObject(smart_objects::SmartType_Map);
+
+  (*command_msg)[strings::msg_params][strings::system_capabilities]
+                [strings::video_streaming_capability]["preferredResolution"] =
+                    smart_objects::SmartObject(smart_objects::SmartType_Map);
```

Comment:
I think I've done in https://github.com/smartdevicelink/sdl_core/pull/1700/commits/7cf5bc32ed0119653513e0a8923154d262f8bde7.